examples/images_contours_and_fields/interpolation_none_vs_nearest.py

```diff
@@ -1,62 +0,0 @@
-"""
```

Comment:
You mean `interpolation_methods.py`?  Right now it certainly doesn't contain all this information.